linux/deploy_vm/rebuild_unattend_install_iso.yml

```diff
@@ -0,0 +1,122 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Rebuild OS ISO image with unattended install file
+# Parameter:
+#   rebuilt_unattend_iso_path: Local path to the rebuilt ISO image with unattend install config file
+#
+- name: "Set fact of the absolute path of source ISO file on ESXi server"
+  set_fact:
+    iso_file_path_on_esxi: "{{ os_installation_iso_list[0] | replace('[', '/vmfs/volumes/') | replace('] ', '/') }}"
+
+# Fetch ISO file from ESXi datastore
+- name: "Download ISO image from ESXi server datastore"
```

Comment:
And this:
Our purpose is reducing and not use tasks delegate_to ESXi host, how about use URL like this "https://xx.xx.xx.xx/folder/test_file.log?dsName=datastore2&dcPath=ha-datacenter" to download the file?

Reply:
`common/esxi_download_datastore_file.yml` was added to download datastore file from URL.